### PR TITLE
Add 0.0.5-SNAPSHOT changelog

### PR DESCRIPTION
**Pull request by xorye**
Signed-off-by: David Kwon <dakwon@redhat.com>

### CODE REVIEW

File: microprofile.ls/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,37 @@
 # Quarkus Language Server Changelog
```
Comment (xorye) [inline microprofile.ls/CHANGELOG.md]: @fbricon change to "MicroProfile Language Server Changelog"?

**Comment by xorye** [comment]
image

File: microprofile.ls/CHANGELOG.md (hunk 1 above)
Comment (fbricon) [reply microprofile.ls/CHANGELOG.md]: yes

File: microprofile.ls/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,37 @@
...
 
+## 0.0.5-SNAPSHOT (February 20, 2019)
+
+### Enhancements
+
+ * Compute deployment JAR only when dependencies changed.. See [#202](https://github.com/redhat-developer/quarkus-ls/pull/202)
+ * Support ConfigProperties for 1.2. See [#200](https://github.com/redhat-developer/quarkus-ls/issues/200)
```
Comment (fbricon) [inline microprofile.ls/CHANGELOG.md]: for Quarkus 1.2

**Comment by fbricon** [comment]
remove -SNAPSHOT from all versions

File: microprofile.ls/CHANGELOG.md (hunk 2 above)
Comment (xorye) [reply microprofile.ls/CHANGELOG.md]: fixed

**Comment by xorye** [comment]
removed

**Comment by xorye** [comment]
image

File: microprofile.ls/CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,38 @@
...
+ * Download transitive dependencies of deployment JAR. See [#179](https://github.com/redhat-developer/quarkus-ls/pull/179)
+ * Missing enum value completion/validation for optional enum. See [#175](https://github.com/redhat-developer/quarkus-ls/issues/175)
+ * Too many microprofile/projectInfo calls. See [#174](https://github.com/redhat-developer/quarkus-ls/issues/174)
+ * Hovering over equals sign is being treated as hovering over the property key. See [#172](https://github.com/redhat-developer/quarkus-ls/issues/172)
+ * CodeLens provider not enabled with LSP4E. See [#156](https://github.com/redhat-developer/quarkus-ls/issues/156)
+
+### Others
+
+ * Fix JavaDoc issues. See [#185](https://github.com/redhat-developer/quarkus-ls/issues/185)
+ * Upversion to 0.0.5-SNAPSHOT. See [#149](https://github.com/redhat-developer/quarkus-ls/pull/149)
```
Comment (fbricon) [inline microprofile.ls/CHANGELOG.md]: please remove

File: microprofile.ls/CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,38 @@
...
+ * Investigate quarkus codelenses for rest endpoints. See [#115](https://github.com/redhat-developer/quarkus-ls/issues/115)
```
Comment (fbricon) [inline microprofile.ls/CHANGELOG.md]: s/Investigate/Display

move to enhancements

File: microprofile.ls/CHANGELOG.md [hunk 5]
```diff
@@ -1,6 +1,38 @@
...
+ * Value from Java enum are not correct. See [#198](https://github.com/redhat-developer/quarkus-ls/issues/198)
+ * Download transitive dependencies of deployment JAR. See [#179](https://github.com/redhat-developer/quarkus-ls/pull/179)
+ * Missing enum value completion/validation for optional enum. See [#175](https://github.com/redhat-developer/quarkus-ls/issues/175)
+ * Too many microprofile/projectInfo calls. See [#174](https://github.com/redhat-developer/quarkus-ls/issues/174)
+ * Hovering over equals sign is being treated as hovering over the property key. See [#172](https://github.com/redhat-developer/quarkus-ls/issues/172)
+ * CodeLens provider not enabled with LSP4E. See [#156](https://github.com/redhat-developer/quarkus-ls/issues/156)
+
+### Others
+
+ * Fix JavaDoc issues. See [#185](https://github.com/redhat-developer/quarkus-ls/issues/185)
```
Comment (fbricon) [inline microprofile.ls/CHANGELOG.md]: move to bug

**Comment by xorye** [comment]
updated

File: microprofile.ls/CHANGELOG.md [hunk 6]
```diff
@@ -1,6 +1,34 @@
-# Quarkus Language Server Changelog
+# MicroProfile Language Server Changelog
 
-## 0.0.4-SNAPSHOT (November 20, 2019)
+## 0.0.5 (February 20, 2019)
+
+### Enhancements
+
+ * Compute deployment JAR only when dependencies changed.. See [#202](https://github.com/redhat-developer/quarkus-ls/pull/202)
```
Comment (fbricon) [inline microprofile.ls/CHANGELOG.md]: remove extra .

**Comment by fbricon** [comment]
I think we should mention the quarkus language server was refactored into microprofile LS

**Comment by xorye** [comment]
@fbricon how's this?
image

**Comment by xorye** [comment]
@fbricon how's this?
image

**Comment by xorye** [comment]
Update:
image

**Comment by fbricon** [comment]
+1

**Comment by xorye** [comment]
I pushed the latest change

**Pull request merged**